chronograf.go

```diff
@@ -762,6 +759,40 @@ type ConfigStore interface {
 	Update(context.Context, *Config) error
 }
 
+// OrganizationConfig is the organization config for parameters that can
+// be set via API, with different sections, such as LogViewer
+type OrganizationConfig struct {
+	OrganizationID string          `json:"organization"`
+	LogViewer      LogViewerConfig `json:"logViewer"`
+}
+
+// LogViewerConfig is the configuration settings for the Log Viewer UI
+type LogViewerConfig struct {
+	Columns []LogViewerColumn `json:"columns"`
+}
+
+// LogViewerColumn is a specific column of the Log Viewer UI
+type LogViewerColumn struct {
+	Name      string           `json:"name"`
+	Position  int32            `json:"position"`
```

Comment:
What is the reasoning behind this being an int32 instead of a regular int?

Reply:
Good catch -- int32 was most consistent with what's used in the other struct in this file, but int would be easier to maintain and we'll keep this is mind for next time